[WIP] Foldable devices with .NET MAUI

conceptdev opened the pull request:
Initial proposal for the **Foldable** NuGet docs.

- [x] What is the right location in the TOC? Currently in **Fundamentals** but could belong in **User Interface > Views** ?
- [ ] ~Whether to include `DualScreenInfo`?~
- [ ] Whether to include `Triggers`?
- [ ] More samples required?

conceptdev committed:
Foldable initial content

conceptdev committed:
fix OPS warnings

conceptdev committed:
fix OPS warning

conceptdev committed:
Foldable add graphic

mahab339 committed:
Typo fix - update to GraphicsView article (#994)

Change (`Center`, of type `PointF`. which specifies...) to (...of type `PointF`, which...).

davidbritch committed:
Update code. (#998)

davidbritch commented on docs/fundamentals/foldable/twopaneview.md:
```diff
@@ -0,0 +1,76 @@
+---
+title: "TwoPaneView foldable and large-screen layout control"
+description: "Learn how to use the SemanticProperties class in a .NET MAUI app, so that a screen reader can speak about the user interface elements on a page."
```
Update required :)

davidbritch commented on docs/fundamentals/foldable/twopaneview.md:
```diff
@@ -0,0 +1,76 @@
+
+The `TwoPaneView` class represents a container with two views that size and position content in the available space, either side-by-side or top-to-bottom. `TwoPaneView` inherits from `Grid` so the easiest way to think about these properties is as if they are being applied to a grid.
+
+![Surface Duo dual-screen emulator showing a basic TwoPaneView test app](media/emulator-foldable-maui-app.png)
```
Afraid we need to use the `:::image` format now. See other docs for examples.

davidbritch reviewed (commented):
Maybe add your alias/github to the metadata, so you get to see any issues opened by users.

conceptdev committed:
Foldable content move in TOC

conceptdev committed:
Foldable update image syntax

conceptdev committed:
Merge branch 'net7' into conceptdev-605-foldable

conceptdev committed:
"Files should end with a single newline character"

conceptdev committed:
Merge branch 'conceptdev-605-foldable' of https://github.com/dotnet/docs-maui into conceptdev-605-foldable

conceptdev committed:
Foldable links to dual-screen docs

adegeo committed:
First pass at some more in-depth articles on publishing win/android (#997)

* initial articles for android/windows

* add android files

* various errors

* minor fixes

* Apply suggestions from code review

Co-authored-by: David Britch <davidbritch@users.noreply.github.com>

* fix svg

* Update docs/android/deployment/overview.md

Co-authored-by: David Britch <davidbritch@users.noreply.github.com>

Co-authored-by: David Britch <davidbritch@users.noreply.github.com>

mahab339 committed:
Typo fix - update to geometries article (#1005)

- Change `SweepDirection` enumeration members description to reflect corresponding values.
- Change  (... type `GeometryCollection`, which species the objects that define...) to (.. type `GeometryCollection`, which specifies...).

davidbritch committed:
Add warning about null return value. (#1009)

mahab339 committed:
Typo fix - update to shapes article (#1008)

Change (.... also have a `StrokeDashOffset` property ,...) to (... have a `StrokeDashOffset` property, ...).

mahab339 committed:
Typo fix - update to path markup syntax article (#1007)

- Change (`-Infinity` represents `double.NegativeInfinity`.) to (`-Infinity` represents...). 
- Change (... two specified control point.) to (...  two specified control points.)

mahab339 committed:
Fix to error produced from example - update to image article (#1006)

Change (CacheValidity="10:00:00.0") to (CacheValidity="10:00:00"). 
Reason: extra zero produces error:  "The TimeSpan could not be parsed because at least one of the numeric components is out of range or contains too many digits.". 
Project workload: .NET Maui 7.0.0.

adegeo committed:
Add checking for emulator updates to troubleshooting (#1003)

* Update troubleshooting.md

* Update troubleshooting.md

tiwahu committed:
Update image.md (#1014)

* Update image.md

Fix XAML example of a 10-day TimeSpan.

* Update image.md

Co-authored-by: David Britch <davidbritch@users.noreply.github.com>

davidbritch committed:
versionName is in the manifest node (#1028)

davidbritch committed:
iOS 11 min requirement using Xcode 14 (#1029)

davidbritch committed:
Remove limitations because OpenGLView doesn't exist in .NET MAUI (#1032)

davidbritch committed:
Add Mac instructions. (#1034)

davidbritch committed:
VSMac instructions for live visual tree (#1035)

* Add Mac coverage.

* Edits.

davidbritch committed:
Add .NET header (#1039)

mahab339 committed:
Typo fix - update to editor article (#1040)

Change (...enter and edit multiple lines of text.]) to (...multiple lines of text.).

joel-heath committed:
Update blazorwebview.md (#1046)

Added some missing double quotes in the ContentPage tag.

sigged committed:
Add necessary overrides in MainActivity (#1051)

* Add necessary overrides in MainActivity

* Edit text to correctly reference code snippet.

* Edit.

Co-authored-by: David Britch <davidbritch@users.noreply.github.com>

davidbritch committed:
What's new for Oct (#1054)

conceptdev committed:
oops remove Xamarin.Forms mention

conceptdev committed:
Merge branch 'main' into conceptdev-605-foldable

conceptdev committed:
Revert "Update image.md (#1014)"

This reverts commit 4868c57e6ba0ff2742b444de6df29f22f261362c.

conceptdev committed:
Revert "versionName is in the manifest node (#1028)"

This reverts commit 19b3aadbb7ea6bdad9e9a59fff5c07020b91e041.

conceptdev committed:
Revert "iOS 11 min requirement using Xcode 14 (#1029)"

This reverts commit 07c93dcc3272a7bf2c9f484f08faa5fd1d0c69fe.

conceptdev committed:
Revert "Remove limitations because OpenGLView doesn't exist in .NET MAUI (#1032)"

This reverts commit 38d1fe0b88a823ca3443ff30c63016ad4f458291.

conceptdev committed:
Revert "Add Mac instructions. (#1034)"

This reverts commit 0b5f83e37585ba340be2103cd6727bc1f430cdc6.

conceptdev committed:
Revert "VSMac instructions for live visual tree (#1035)"

This reverts commit ebf612e97d2605415f9f5cc8a383aaf5712821a4.

conceptdev committed:
Revert "Add .NET header (#1039)"

This reverts commit 3ef32d55d94cd92b55cbff4f3c70ea6e35cf464e.

conceptdev committed:
Revert "Typo fix - update to editor article (#1040)"

This reverts commit 484057633d6d42f1dfb942606ab63936819ec7d3.

conceptdev committed:
Revert "Update blazorwebview.md (#1046)"

This reverts commit e378bb041f50c5fe31857b5af1b3c08c73a16ea6.

conceptdev committed:
Revert "Add necessary overrides in MainActivity (#1051)"

This reverts commit 3152248406767ea15daa5e0234daad1efbc6e34f.

conceptdev committed:
Revert "What's new for Oct (#1054)"

This reverts commit 8a08967cfeda1d67195e791b94f5ada5040c0f6a.

conceptdev commented:
Screwed up the branch, moved to #1057
